data/overlays/actors/z_boss_ganon2.data.s

```diff
@@ -124,7 +124,7 @@ glabel D_80908360
 glabel D_80908960
- .word 0x4E200FA0, 0x00000000, 0x04000000, 0xFFFFFFFF, 0x4E20F060, 0x00000000, 0x04000400, 0xFFFFFFFF, 0x46500FA0, 0x00000000, 0x039A0000, 0xFFFFFFFF, 0x4650F060, 0x00000000, 0x039A0400, 0xFFFFFFFF, 0x3E800FA0, 0x00000000, 0x03330000, 0xFFFFFFFF, 0x3E80F060, 0x00000000, 0x03330400, 0xFFFFFFFF, 0x36B00FA0, 0x00000000, 0x02CD0000, 0xFFFFFFFF, 0x36B0F060, 0x00000000, 0x02CD0400, 0xFFFFFFFF, 0x2EE00FA0, 0x00000000, 0x02660000, 0xFFFFFFFF, 0x2EE0F060, 0x00000000, 0x02660400, 0xFFFFFFFF, 0x27100FA0, 0x00000000, 0x02000000, 0xFFFFFFFF, 0x2710F060, 0x00000000, 0x02000400, 0xFFFFFFFF, 0x1F400FA0, 0x00000000, 0x019A0000, 0xFFFFFFFF, 0x1F40F060, 0x00000000, 0x019A0400, 0xFFFFFFFF, 0x17700FA0, 0x00000000, 0x01330000, 0xFFFFFFFF, 0x1770F060, 0x00000000, 0x01330400, 0xFFFFFFFF, 0x0FA00FA0, 0x00000000, 0x00CD0000, 0xFFFFFFFF, 0x0FA0F060, 0x00000000, 0x00CD0400, 0xFFFFFFFF, 0x07D00FA0, 0x00000000, 0x00660000, 0xFFFFFFFF, 0x07D0F060, 0x00000000, 0x00660400, 0xFFFFFFFF, 0x00000FA0, 0x00000000, 0x00000000, 0xFFFFFFFF, 0x0000F060, 0x00000000, 0x00000400, 0xFFFFFFFF
+ .word 0x4E200FA0, 0x00000000, 0x04000000, 0xFFFFFFFF, 0x4E20F060, 0x00000000, 0x04000400, 0xFFFFFFFF, 0x46500FA0, 0x00000000, 0x039A0000, 0xFFFFFFFF, 0x4650F060, 0x00000000, 0x039A0400, 0xFFFFFFFF, 0x3E800FA0, 0x00000000, 0x03330000, 0xFFFFFFFF, 0x3E80F060, 0x00000000, 0x03330400, 0xFFFFFFFF, 0x36B00FA0, 0x00000000, 0x02CD0000, 0xFFFFFFFF, 0x36B0F060, 0x00000000, 0x02CD0400, 0xFFFFFFFF, 0x2EE00FA0, 0x00000000, 0x02660000, 0xFFFFFFFF, 0x2EE0F060, 0x00000000, 0x02660400, 0xFFFFFFFF, 0x27100FA0, 0x00000000, 0x02000000, 0xFFFFFFFF, 0x2710F060, 0x00000000, gHeartFullTex, 0xFFFFFFFF, 0x1F400FA0, 0x00000000, 0x019A0000, 0xFFFFFFFF, 0x1F40F060, 0x00000000, 0x019A0400, 0xFFFFFFFF, 0x17700FA0, 0x00000000, 0x01330000, 0xFFFFFFFF, 0x1770F060, 0x00000000, 0x01330400, 0xFFFFFFFF, 0x0FA00FA0, 0x00000000, 0x00CD0000, 0xFFFFFFFF, 0x0FA0F060, 0x00000000, 0x00CD0400, 0xFFFFFFFF, 0x07D00FA0, 0x00000000, 0x00660000, 0xFFFFFFFF, 0x07D0F060, 0x00000000, 0x00660400, 0xFFFFFFFF, 0x00000FA0, 0x00000000, 0x00000000, 0xFFFFFFFF, 0x0000F060, 0x00000000, 0x00000400, 0xFFFFFFFF
```

Comment:
Isn't this supposed to be vertices or something? I highly doubt the 0x02000400 in there is gHeartFullTex

Reply:
just had a look at ganon2 since sirius extracted the overlay data in his branch, and it is indeed just verts. Find and replace mistake prob